Sources/ComposableArchitecture/Effects/Cancellation.swift

```diff
@@ -51,45 +51,44 @@ extension Effect {
       cancellablesLock.lock()
       defer { cancellablesLock.unlock() }
 
-      let subject = PublishSubject<Output>()
-      var values: [Output] = []
-      var isCaching = true
```

Comment:
IIRC this caching stuff was needed for some reason with our examReducer

Reply:
Hmm let me check it out in Helios.

Reply:
Seems to all be working in Helios. No reentrancy concerns. Effects that should cancel do cancel.

Reply:
Recording timers etc all working. Uploads.